ajax/libs/jquery.mb.YTPlayer/package.json

```diff
@@ -34,5 +34,5 @@
     "type": "git",
     "url": "git://github.com/pupunzi/jquery.mb.YTPlayer.git"
   },
-  "license": "MIT"
+  "license": "(MIT OR 	GPL-2.0-or-later)"
```

Comment:
Please see the example https://github.com/cdnjs/cdnjs/pull/13154#discussion_r241965666, thanks.